distribution/ddf-common/src/main/resources/etc/artemis.xml

```diff
@@ -0,0 +1,70 @@
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<configuration xmlns="urn:activemq"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation="urn:activemq /schema/artemis-configuration.xsd">
+
+    <jms xmlns="urn:activemq:jms">
+        <queue name="DLQ"/>
+        <queue name="ExpiryQueue"/>
+    </jms>
+
+    <core xmlns="urn:activemq:core">
+
+        <!-- this could be ASYNCIO or NIO -->
+        <journal-type>NIO</journal-type>
+        <paging-directory>./data/paging</paging-directory>
+        <bindings-directory>./data/bindings</bindings-directory>
+        <journal-directory>./data/journal</journal-directory>
+        <large-messages-directory>./data/large-messages</large-messages-directory>
+        <journal-min-files>10</journal-min-files>
+
+        <!--
+         This value was determined through a calculation.
+         Your system could perform 0.63 writes per millisecond
+         on the current journal configuration.
+         That translates as a sync write every 1591999 nanoseconds
+        -->
+        <journal-buffer-timeout>1591999</journal-buffer-timeout>
```

Comment:
Is this machine-dependent? Is there a way to test on a production system and get a different empirical value for to set here?